### PR TITLE
add Landsat pan band (08) values in histogramm min/max fetch

### PR DESCRIPTION
**Pull request by vincentsarago**
This PR adds Landsat8 Panchromatic min/max fetch when getting Landsat metadata.

cc @sgillies @jacquestardie

### CODE REVIEW

File: rio_tiler/landsat8.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def metadata(sceneid, pmin=2, pmax=98):
     info = {'sceneid': sceneid}
     info['bounds'] = toa_utils._get_bounds_from_metadata(meta_data['PRODUCT_METADATA'])
 
-    bands = ['1', '2', '3', '4', '5', '6', '7', '9', '10', '11']
+    bands = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11']
```
Comment (sgillies) [inline rio_tiler/landsat8.py]: @vincentsarago will you add a note about this to the change log?

**Comment by vincentsarago** [comment]
thanks @sgillies

File: CHANGES.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+Next
+----
+- fetch Landsat panchromatic band min/max (#58)
+
```
Comment (sgillies) [inline CHANGES.txt]: The bug fix here is that band 8 was missing, yes? Let's say so in the change log.

Sorry to be a pain!

Comment (vincentsarago) [reply CHANGES.txt]: I'll re-phrase

**Pull request merged**